standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1467,7 +1467,7 @@ public void replTableWriteIdState(ReplTblWriteIdStateRequest rqst) throws MetaEx
 
     // Schedule Major compaction on all the partitions/table to clean aborted data
     if (numAbortedWrites > 0) {
-      CompactionRequest compactRqst = new CompactionRequest(rqst.getDbName(), rqst.getTableName(),
+      CompactionRequest compactRqst = new CompactionRequest(dbName, tblName,
```

Comment:
Why do we need this change here?

Reply:
we need the database name and table name to be in lowercase.

Reply:
Fine.

Reply:
Ok.